main.js

```diff
@@ -22,3 +22,70 @@ navItems.forEach((navItem) => {
     sidebar.classList.toggle('show-sidebar');
   });
 });
+
+const popupContent = {
+  headerMobile: 'Multi Post Stories',
+  headerDesktop: 'Keeping track of hundreds  of components website',
+  closeIcon: './images/times.svg',
+  descriptionDesktop: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+  descriptionMobile: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+  timeFont: './images/times.svg',
+  featuredImage: './images/popupImg.svg',
+  liveVersion: 'https://olamarx.github.io/Olamarx-portfolio/',
+  sourceLink: 'https://github.com/Olamarx/Olamarx-portfolio',
+  githubfont: '<img src="./images/githubfont.svg" alt="">',
+  fontLive: '<img src="./images/livepreview.svg" alt="Live-preview">',
+
+  buttons: {
+    html: 'HTML',
+    bootstrap: 'Bootstrap',
+    ruby: 'Ruby on rails',
+  },
+};
```

Comment:
It's crucial that your JS code implements an array of objects to store JavaScript objects containing the information for each project. The idea behind it is for you to handle dynamic information and show it in different HTML elements. Right now, you only have one object that populates every popup for every project and that is pretty static. Take a look at the following: 


    const projects = [{
        name: "Submarine game",
        description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
        image: "link_to_image.jpg",
        technologies: ["Phaser3", "JavaScript", "Jest"],
        link: "link_to_live_version.com",
    },
    {
        name: "Submarine game 2",
        description: "Another desc.",
        image: "link_to_image_2.jpg",
        technologies: ["Phaser4", "JS", "Example"],
        link: "link_to_live_version_2.com",
    }
    ]

With this structure, it will be super easy for you to add or remove projects from your portfolio because you will only modify this data structure and access it properties in a loop with `projects[i].name`